src/js/components/search-filter/SearchFilter.jsx

```diff
@@ -4,12 +4,27 @@ import './searchfilter.style.less';
 
 class SearchFilter extends Component {
 
+  state = {
+    searchby: this.props.searchby
+  }
+
+  changeHandler = () => {
+    if (event.target.value && event.target.value !== this.state.searchby) {
```

Comment:
here you are relying on MouseEvent which actually is not a best idea here to use - especially that you're ing a react-bootstrap that is providing a nice API - so the best wat would be usage as documentation is saying so

```
  changeHandler = (filter) => {
    if (filter !== this.state.searchby) {
      this.setState({searchby: filter}, () => this.props.changeHandler(this.state.searchby));
    }
  };
```

from this point we can improve our method:

1) as you have a value immediately you don't need to wait for state to be set especially as you're already cheking if value was changed to trigger handler from props, so you can take this outside of callback function and write right after setting state with a value from changeHandler

2) and mirroring which is great thing but need a bit of practice ;)
```
changeHandler = (searchby) => {
    if (searchby !== this.state.searchby) {
      this.setState({searchby});
      ...[rest of code]
    }
  };
```